solr/core/src/java/org/apache/solr/handler/StreamHandler.java

```diff
@@ -199,10 +200,16 @@ public void inform(SolrCore core) {
       .withFunctionName("mult", MultiplyEvaluator.class)
       .withFunctionName("sub", SubtractEvaluator.class)
       .withFunctionName("log", NaturalLogEvaluator.class)
+
       // Conditional Stream Evaluators
       .withFunctionName("if", IfThenElseEvaluator.class)
       ;
 
+      // Date evaluators
```

Comment:
Yeah, I agree - maybe we should take a look at that assumption of 1 class == 1 function and re-evaluate if there's a better way.